# Generated by Django 4.0.6 on 2022-10-31 12:28

from django.db import migrations


def track_new_email_settings(apps, schema_editor):
    try:
        from common.klaviyo.util import track_email_settings
        CustomUser = apps.get_model('account.CustomUser')
        for user in CustomUser.objects.all():
            track_email_settings(user_id=user.id)
    except Exception:
        print('migration account.0024 is failed but nothing to worry about')


class Migration(migrations.Migration):

    dependencies = [
        ('account', '0023_remove_emailsettings_alerts_city_and_more'),
    ]

    operations = [
        migrations.RunPython(track_new_email_settings)
    ]
